docs/introduction/bigpicture.rst

```diff
@@ -1,7 +1,7 @@
 Big Picture
 ===========
 
-Ocelot is aimed at people using .NET running a micro services / service orientated architecture that need a unified point of entry into their system.
+Ocelot is aimed at people using .NET running a micro services / service oriented architecture that need a unified point of entry into their system.

```

Comment:
not `a micro services` but `a microservices`

not `service oriented` but `service-oriented`